cmake/z3.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ba3ec7bce..7ca1894fd 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -446,13 +446,13 @@ configure_file(
+ 
+ # Target needs to be declared before the components so that they can add
+ # dependencies to this target so they can run their own custom uninstall rules.
+-add_custom_target(uninstall
+-  COMMAND
+-  "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
+-  COMMENT "Uninstalling..."
+-  USES_TERMINAL
+-  VERBATIM
+-)
++# add_custom_target(uninstall
++#   COMMAND
++#   "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
++#   COMMENT "Uninstalling..."
++#   USES_TERMINAL
++#   VERBATIM
++# )
```

Comment:
Why do we need to disable this?

Reply:
The `install` target conflicts with P4C's `install` target. The problem is that CMake does not have namespaces.